Moderation-Policy.md

```diff
@@ -0,0 +1,100 @@
+## Moderation policy
+
```

Comment:
We should add a very quick TLDR here something akin to

> if you are not a member of the node.js org please send moderation request to  report@nodejs.org or ... (other communication methods)

or something to that effect